indico/web/assets/bundles.py

```diff
@@ -240,6 +240,7 @@ def rjs_bundle(name, *files):
     'js/lib/jquery.multiselect.js',
     'js/lib/jquery.multiselect.filter.js',
     'js/jquery/jquery-migrate-silencer.js' if not Config.getInstance().getDebug() else None] +
+    ['js/lib/jquery.tablesorter.js'] +
```

Comment:
I have finally found the cause of the problem. Creating a custom widget in `Dragndrop.js` conflicts with `tablesorter` defined in jquery.tablesorter.js. However, simply changing the name of the widget may lead to backward compatibility problems.